src/routes/blog-route/article-route.jsx

```diff
@@ -4,14 +4,14 @@ import ArticlePage from "../../page/blog/article-page.jsx";
 export function ArticleRoute({ type }) {
   return (
     <>
-      <div className="relative">
-        <Header>
-          <Header.Logo />
-          <Header.NavLinks />
-          <Header.Profile />
+      <div className="relative" data-test-id="article-header">
+        <Header data-test-id="article-header">
+          <Header.Logo data-test-id="article-header-logo" />
+          <Header.NavLinks data-test-id="article-header-navlinks" />
+          <Header.Profile data-test-id="article-header-profile" />
```

Comment:
```suggestion
          <Header.Logo />
          <Header.NavLinks />
          <Header.Profile />
```
They don't apply through for some reason. Thankfully we can tell based on the color syntax highlighting.